samples/05_content_publishers/html_table_to_pandas_data_frame_to_portal_item.ipynb

```diff
@@ -534,46 +490,57 @@
+       "3     New Caledonia              42.5   Oceania        Melanesia   \n",
+       "4            Serbia              39.1    Europe  Southern Europe   \n",
        "\n",
-       "  Region[3]     Subregion[3]  Population 2017  \\\n",
-       "1  Americas    North America      326474000.0   \n",
-       "2  Americas    South America           3000.0   \n",
-       "3      Asia     Western Asia       28120000.0   \n",
-       "4   Oceania        Melanesia         270000.0   \n",
-       "5    Europe  Southern Europe        6946000.0   \n",
+       "   Population_2017 Civilian_firearms  Computation_method  Registered_firearms  \\\n",
+       "0        326474000         393347000                   1            1073743.0   \n",
+       "1             3000              2000                   2               1705.0   \n",
+       "2         28120000          14859000                   2                  NaN   \n",
+       "3           270000            115000                   2              55000.0   \n",
+       "4          6946000           2719000                   2            1186086.0   \n",
        "\n",
-       "  Estimate of firearms in civilian possession  Computation method  \\\n",
-       "1                                   393347000                 1.0   \n",
-       "2                                        2000                 2.0   \n",
-       "3                                    14859000                 2.0   \n",
-       "4                                      115000                 2.0   \n",
-       "5                                     2719000                 2.0   \n",
-       "\n",
-       "   Registered firearms Unregistered firearms  \n",
-       "1            1073743.0      392,273,257 Est.  \n",
-       "2               1705.0                   295  \n",
-       "3                  NaN                   NaN  \n",
-       "4              55000.0                 60000  \n",
-       "5            1186086.0               1532914  "
+       "  Unregistered_firearms  \n",
+       "0      392,273,257 Est.  \n",
+       "1                   295  \n",
+       "2                   NaN  \n",
+       "3                 60000  \n",
+       "4               1532914  "
       ]
      },
-     "execution_count": 9,
+     "execution_count": 34,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "converted_column = pd.to_numeric(df[\"Estimate of civilian firearms per 100 persons\"], errors = 'coerce')\n",
-    "df['Estimate of civilian firearms per 100 persons'] = converted_column\n",
     "df.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gis = GIS(profile=\"your_online_admin_profile\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dpath = r\"/Users/john3092/Job/\"\n",
```

Comment:
probably ok, but this exposes your local path

Reply:
Thanks. I think it was from an unrendered widget. When I cleared that output, no more exposure!